src/videojs-contrib-hls.js

```diff
@@ -273,6 +273,7 @@ class HlsHandler extends Component {
     if (tech.options_ && tech.options_.playerId) {
       let _player = videojs(tech.options_.playerId);
 
+      _player.gapSkipper({tech});
```

Comment:
we should move this line outside of this if check, maybe even into the src function of HlsHandler